LLama/OldVersion/Utils.cs

```diff
@@ -82,5 +83,16 @@ public static unsafe string PtrToStringUTF8(IntPtr ptr)
             return Encoding.UTF8.GetString(bytes.ToArray());
 #endif
         }
+
+        /// <summary>
+        /// Converts a bool "value" to a signed byte of "1" for true and "0" for false to be compatible with a 1 byte C-style bool.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        public static sbyte BoolToSignedByte(bool value)
+        {
+            return value ? SbyteBoolArray[1] : SbyteBoolArray[0];
```

Comment:
```suggestion
            return value ? (sbyte)0 : (sbyte)1;
```

Reply:
I just noticed this is in `OldVersion.Utils`, so this could probably just use the version in the other Utils file instead.